src/main/java/com/google/sps/data/BusinessesService.java

```diff
@@ -18,23 +18,60 @@
 import com.google.maps.model.PlacesSearchResponse;
 import com.google.maps.model.PlacesSearchResult;
 import java.util.logging.Logger;
+import java.util.Iterator;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 
 /** BusinessesService object representing all businesses 
 * components of the webapp.
  **/
 public class BusinessesService {
   
-  private List<Listing> businesses;
+  private List<Listing> allBusinesses;
   private final String KEY = "REDACTED";
   private final static Logger LOGGER = 
         Logger.getLogger(BusinessesService.class.getName());
   private final int ALLOWED_SEARCH_REQUESTS = 3;
 
   /** Create a new Businesses instance
-  * @param businesses businesses from SmallCityService
+  * @param allBusinesses businesses from SmallCityService
   **/
-  public BusinessesService(List<Listing> businesses) {
-    this.businesses = businesses;
+  
+  public BusinessesService(List<Listing> allBusinesses) {
+    this.allBusinesses = allBusinesses;
+  }
+
+  public PreparedQuery getBigBusinessFromDatabase(){
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query query = new Query("BigBusinesses");
+    PreparedQuery queryOfDatabase = datastore.prepare(query);
+    return queryOfDatabase;
+  }
+
+  public List<Listing> removeBigBusinessesFromResults(PreparedQuery queryOfDatabase){
+    Iterator<Listing> smallBusinesses =  allBusinesses.iterator();
```

Comment:
Feel free to leave this as "businesses". The other variable was renamed.

Reply:
fixed